src/pylexibank/commands/language_profiles.py

```diff
@@ -0,0 +1,56 @@
+"""
+Create a profile for individual languages from an already prepared general profile.
+"""
+from cldfbench.cli_util import get_dataset
+from clldutils.clilib import ParserError
+from pycldf import Dataset
+
+from pylexibank.cli_util import add_dataset_spec
+from pathlib import Path
+from collections import defaultdict
+from pylexibank import progressbar
+from csvw.dsv import UnicodeDictReader
+
+
+def register(parser):
+    add_dataset_spec(parser)
+    parser.add_argument(
+        '-f', '--force',
+        action='store_true',
+        help='Overwrite existing profile',
+        default=False)
+
+
+def run(args):
+
+    ds = get_dataset(args)
+    wordlist = Dataset.from_metadata(ds.cldf_dir / "cldf-metadata.json")
+    p = ds.etc_dir / "orthography.tsv"
+    if not p.exists():
+        raise ParserError("profile does not exist but is needed for creation")
+    with UnicodeDictReader(p, delimiter="\t") as reader:
+        profile = {}
+        for row in reader:
+            profile[row["Grapheme"]] = row["IPA"]
+
+    
+    for language in progressbar(
+            wordlist.objects("LanguageTable"), 
+            desc="creating profiles"):
+        data = defaultdict(int)
+        for form in language.forms:
+            for grapheme in form.data["Graphemes"].split():
```

Comment:
This throws an error in datasets where the `Graphemes` column is empty (e.g. [hillburmish](https://github.com/lexibank/hillburmish)):

      File "[...]/pylexibank/src/pylexibank/commands/language_profiles.py", line 42, in run
        for grapheme in form.data["Graphemes"].split():
    AttributeError: 'NoneType' object has no attribute 'split'